lib/dwolla.ex

```diff
@@ -86,10 +86,11 @@ defmodule Dwolla do
   end
 
   def process_response_body(""), do: ""
+
   def process_response_body(body) do
-    case Poison.Parser.parse(body) do
+    case Poison.decode(body) do
```

Comment:
@wfgilman Thanks for the callout, updated this to `decode` as this seems to replace the behavior of `Poison.Parser.parse` in v4